pkg/util/randutil/rand.go

```diff
@@ -0,0 +1,9 @@
+package randutil
+
+import "crypto/rand"
+
+// Read reads len(p) bytes from the rand.Reader
+// This is a helper function to wrap crypto/rand.Read
+func Read(p []byte) (n int, err error) {
```

Comment:
https://github.com/grafana/tempo/pull/2079#issuecomment-1423527005

Because of this.
But suddenly I realized that I could just use package alias. I'll update this.

Reply:
fixed.